rllib/core/models/base.py

```diff
@@ -301,11 +299,11 @@ def build(self, framework: str):
 
     @override(Model)
     def get_input_specs(self) -> Optional[Spec]:
-        return convert_to_canonical_format([SampleBatch.OBS, STATE_IN])
+        return [SampleBatch.OBS]
 
     @override(Model)
     def get_output_specs(self) -> Optional[Spec]:
-        return convert_to_canonical_format([ENCODER_OUT, STATE_OUT])
+        return [ACTOR, CRITIC]
```

Comment:
should this not be empty? cuz why do you assume that encoder will have two heads: 1. actor and 2. critic?

Reply:
We are also overriding this in the ActorCritic and StateFul actor critic instantiations.